_episodes/02-enter-sandman.md

```diff
@@ -93,9 +93,7 @@ As you've seen above, the exit code from the last executed command is stored in
 
 > ## Snake Charming
 >
-> To enter the Python interpreter, simply type `python` in your command line.
->
-> Once inside the Python interpreter, simply type `exit()` then press enter, to exit.
+> To enter the Python interpreter, simply type `python` in your command line. If you do not have `python`, but rather `python3` then use python3 instead. You will have to then swap `python` with `python3` in the Python code block in the Python section.
```

Comment:
Maybe for brevity: "If you do not have `python`, but rather `python3` then use python3 instead. You will have to then swap `python` with `python3` in the Python code block in the Python section."  --> On some systems, you need to use the `python3` command instead of `python`.

Reply:
Actually let's swap it, so that they use `python3` by default (and fall back to `python`).

Also add that they should check the version of python with `python --version` (because the next example needs them to know which one they have)